src/renderers/common/RenderObject.js

```diff
@@ -191,7 +191,7 @@ export default class RenderObject {
 
 		for ( const property of getKeys( material ) ) {
 
-			if ( /^(is[A-Z]|_)|^(visible|version|uuid|name|opacity|userData)$/.test( property ) ) continue;
+			if ( /^(is[A-Z]|_)|^(visible|version|name|opacity|userData)$/.test( property ) ) continue;
```

Comment:
Instead of always adding the uuid, would it make sense to just do this for instances meshes? 

There is the `if ( object.count > 1 ) {` further down where the check could be added.

Reply:
Indeed, even better!